lua/wire/client/hlzasm/hc_preprocess.lua

```diff
@@ -70,6 +70,54 @@ function HCOMP:ParsePreprocessMacro(lineText,macroPosition)
       CPULib.CPUName = pragmaCommand
     elseif pragmaName == "searchpath" then
       table.insert(self.SearchPaths,pragmaCommand)
+    elseif pragmaName == "allow" or pragmaName == "zap" then
+    if not self.Settings.AutoBusyRegisters then
+      self.Settings.AutoBusyRegisters = true
+    end
+    local Arguments = string.gmatch(macroParameters,'[^, ]+') -- comma separate the two registers
+    Arguments() -- drop first space
+    local StartRegister = trimString(Arguments())
+    local EndRegister = trimString(Arguments())
```

Comment:
This might lua error if Arguments() returns nil. This might be safer to use `local StartRegister, EndRegister = string.match(macroParameters, "([^,%s]+)[,%s]+([^,%s]+)")` and check StartRegister and EndRegister for nil.